resources/metadata/detect.yaml

```diff
@@ -21,7 +21,10 @@ spec:
     params:
       - name: apiToken
```

Comment:
As most steps use `token` as parameter name, you could also align the main parameter name.

Reply:
good point!
Will do so ...

Reply:
done, please re-check